runtime/colors.rs

```diff
@@ -8,12 +8,18 @@ use termcolor::{Ansi, ColorSpec, WriteColor};
 #[cfg(windows)]
 use termcolor::{BufferWriter, ColorChoice};
 
+// checks if the output is piped to a tty
+lazy_static::lazy_static! {
+  static ref IS_TTY: bool = atty::is(atty::Stream::Stdout);
+}
+
 lazy_static::lazy_static! {
   static ref NO_COLOR: bool = std::env::var_os("NO_COLOR").is_some();
 }
```

Comment:
```suggestion
lazy_static::lazy_static! {
  // Checks if the output is piped to a tty.
  static ref IS_TTY: bool = atty::is(atty::Stream::Stdout);
  static ref NO_COLOR: bool = std::env::var_os("NO_COLOR").is_some();
}
```

You can have multiple declarations inside a single `lazy_static` block.